Test.cmd

```diff
@@ -1,28 +1,37 @@
 @if not defined _echo @echo off
 setlocal
 
-if [%1]==[]                GOTO NETFX
-if [%1]==[netfx]           GOTO NETFX
-if [%1]==[netstandard-win] GOTO VNEXT
-if [%1]==[netstandard]     GOTO VNEXT
-if [%1]==[all]             GOTO ALL
-
-:NETFX
-cmd /c "%~dp0src\Test.cmd"
-set exitcode=%errorlevel%
-GOTO END
-
-:VNEXT
-cmd /c "%~dp0vNext\Test.cmd"
-set exitcode=%errorlevel%
-GOTO END
-
-:ALL
-cmd /c "%~dp0src\Test.cmd"
-set exitcode=%errorlevel%
-cmd /c "%~dp0vNext\Test.cmd"
-set /a exitcode=%errorlevel%+%exitcode%
-
-:END
-endlocal&set exitcode=%exitcode%
-exit /B %exitcode%
\ No newline at end of file
+SET CONFIGURATION=Release
+
+SET CMDHOME=%~dp0
+@REM Remove trailing backslash \
+set CMDHOME=%CMDHOME:~0,-1%
+
+pushd "%CMDHOME%"
+@cd
+
+SET TestResultDir=%CMDHOME%\Binaries\%CONFIGURATION%\TestResults
+
+if not exist %TestResultDir% md %TestResultDir%
+
+SET _Directory=bin\%CONFIGURATION%\net461\win10-x64
```

Comment:
is win10-x64 appropriate here? is there a less hardcody approach we could take?
if it works in both CI servers (which apparently does), I don't feel very strongly for now if it's not easy to generalize, since we will probably just replace this with a call to `dotnet test` in the near future